tests/ObjectConfiguration.tex

```diff
@@ -0,0 +1,75 @@
+\section{Replicating a Configuration of Objects}
+
+The robot has to understand relationships between objects, instruct a human to replicate that configuration of objects, and fix errors that the human makes.
+
+\subsection{Focus}
+
+This test focuses on object detection, recognition, and description, as well as goal-oriented human-robot collaboration.
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Robot Starting Room}: any room.
+	\item \textbf{Human Starting Room}: a room adjacent to Robot Starting Room.
+	\item \textbf{Human Collaborator}: The human collaborator is in the Human Starting Room.
+	\item \textbf{Robot Objects} 10 objects are arranged in a space in front of the robot, potentially on top of a table or other furniture.
+	\item \textbf{Human Objects:} In this room there are the identical objects as in the other room, but arranged differently.
+\end{itemize}
+
+\subsection{Task Part 1}
+
```

Comment:
Task requires complete redesign. It is too fine-grained and the goal of each of the steps is underspecified. It is unclear where the collaboration is leading to.

Reply:
By "unclear where the collaboration is leading to" do you mean you would like there to be more of a story justification?

Reply:
The test is extremely hard to follow because there is no intuitive goal. *Serve Breakfast*, *Cocktail Party*, *Follow Me*, *Fetch and Carry*, etc., give you a clue about the task and goal from the very title. Any person can easily grasp the context and what's the ultimate goal. In this test I couldn't find a direct and intuitive application for the task, something that people in the audience can easily relate to something they do in daily life.

The closest thing that came to my mind was when I move all objects from one surface to another while cleaning and then I can't remember what was the original configuration. Here It would be useful if the robot reminds me whether my ancient *Ming pot* was left or right of the stand with the *Scroll of Truth*, stuff that is most valuable to me and the robot is not allowed to touch. However:
- I am in the same room
- I am interacting with exactly the same objects
- Either I ask the robot or the robot realizes the mistake.
Therefore, I see no reason why I should replicate a setup that is in another room, nor why should I receive instructions from anyone (human or robot, that's extremely annoying).

However, I am not sure if what I mentioned above actually has something to do with the proposed test based only in the content of the tex file.

Designing a test is a very difficult task for it needs to be cross-culture understandable, so the first advice is give your test *as is* to read to a 3rd person (better if outside the work group) and ask them to explain back to you. Don't explain, just take notes and ask why that person makes that interpretation.